src/atomate2/vasp/flows/phonons.py

```diff
@@ -0,0 +1,226 @@
+    generate_frequencies_eigenvectors,
+    generate_phonon_displacements,
+    run_phonon_displacements,
+)
+from atomate2.vasp.sets.core import StaticSetGenerator
+
+__all__ = ["PhononMaker"]
+
+
+@dataclass
+class PhononMaker(Maker):
+    """
+    Maker to calculate harmonic phonons with VASP and Phonopy.
+
+    Calculate the harmonic phonons of a material. Initially, a tight structural
+    relaxation is performed to obtain a structure without forces on the atoms.
+    Subsequently, supercells with one displaced atom are generated and accurate
+    forces are computed for these structures. With the help of phonopy, these
+    forces are then converted into a dynamical matrix. To correct for polarization
+    effects, a correction of the dynamical matrix based on BORN charges can
+    be performed.     Finally, phonon densities of states, phonon band structures
+    and thermodynamic properties are computed.
+
+    .. Note::
+        It is heavily recommended to symmetrize the structure before passing it to
+        this flow. Otherwise, a different space group might be detected and too
+        many displacement calculations will be generated.
+        It is recommended to check the convergence parameters here and
+        adjust them if necessary. The default might not be strict enough
+        for your specific case.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    sym_reduce : bool
+        Whether to reduce the number of deformations using symmetry.
+    symprec : float
+        Symmetry precision to use in the reduction of symmetry.
+    displacement: float
+        displacement distance for phonons
+    min_length: float
+        min length of the supercell that will be build
+    conventional: bool
+        if true, the supercell will be built from the conventional
+        cell and all properties will be related
+        to the conventional cell
+    bulk_relax_maker : .BaseVaspMaker or None
+        A maker to perform a tight relaxation on the bulk. Set to ``None`` to skip the
+        bulk relaxation
+    generate_phonon_diplsacements_kwargs: dict
+        keyword arguments paseed to :oj: enerate_phonon_displacements
+    run_phonon_displacements_kwargs : dict
+        Keyword arguments passed to :obj:`run_phonon_displacements__kwargs`.
+    born_maker: .BaseVaspMaker or None
+        Maker to compute the BORN charges.
+    phonon_displacement_maker : .BaseVaspMaker or None
+        Maker used to compute the forces for a supercell.
+    generate_frequencies_eigenvectors_kwargs : dict
+        Keyword arguments passed to :obj:`generate_frequencies_eigenvectors`.
+    """
+
+    # TODO: add some unit conversion factors
+    #  to easily use other codes to compute phonons?
+    name: str = "phonon"
+    sym_reduce: bool = True
+    symprec: float = SETTINGS.SYMPREC
+    displacement: float = 0.01
+    min_length: float = 20.0
+    conventional: bool = False
```

Comment:
Agreed but then we should probably also make the computation of the band structure optional?

Reply:
Potentially? Although it might be possible to use [the AUTO option for PRIMITIVE_AXIS](https://phonopy.github.io/phonopy/setting-tags.html#primitive-axes-or-primitive-axis) to automatically align the band structure whatever input structure you use?

Reply:
Yes, true.

Reply:
Yes, I know this option. Need to check.

Reply:
I would like to be able to use the kpath options from pymatgen. I am not sure phonopy uses the same convention for the primitive cell.